js_modules/dagit/packages/core/src/ui/StickyTableContainer.tsx

```diff
@@ -0,0 +1,45 @@
+import {Colors} from '@dagster-io/ui';
+import styled from 'styled-components/macro';
+
+interface Props {
+  $top: number;
+}
+
+/**
+ * Wrap a `Table` component with this to cause its `thead` to be sticky while scrolling.
+ *
+ * `$top` is the pixel value of the point in the scrolling container that the `thead`
+ * should stick to. Probably `0`.
+ */
+export const StickyTableContainer = styled.div<Props>`
+  thead tr {
+    position: sticky;
+    top: ${({$top}) => $top}px;
+    background-color: ${Colors.White};
+    z-index: 1;
```

Comment:
Oh interesting I didn't realize there was a CSS-only way to do this, this seems great!